content/docs/user-guide/components/reference/terrain/surface-material-list.md

```diff
@@ -5,8 +5,11 @@ description: 'Open 3D Engine (O3DE) Terrain Surface Materials List component ref
 weight: 100
 ---
 
-The **Terrain Surface Materials List** component defines mappings between surface types and render materials.  You can use this to change the appearance of a surface type within different areas of your game. When you assign a material to a surface type, all the visible surfaces of that type, within the bounds of the required [Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape/) component, will adopt that material.
+The **Terrain Surface Materials List** component defines mappings between surface types and render materials. You can use this to change the appearance of a surface type within different areas of your game. When you assign a material to a surface type, all the visible surfaces of that type, within the bounds of the required [Axis Aligned Box Shape](/docs/user-guide/components/reference/shape/axis-aligned-box-shape/) component, will adopt that material.
 
+This component makes use of [macro materials](/docs/user-guide/components/reference/terrain/terrain-macro-material) and [detail materials](/docs/user-guide/components/reference/terrain/terrain-detail-material/) across a large terrain surface and blends between them. This blending of detail materials with the macro material enables you to use small, high fidelity, tiled detail materials by creating variations in color and lighting across the terrain.
+
+For an example of how to use the detail materials with a surface materials list, please follow the [apply detail materials](/docs/learning-guide/tutorials/environments/create-terrain-from-images/#apply-detail-materials) section of the **Create Terrain from Images** tutorial.
```

Comment:
We should also explain in this section a bit more about how blending works. @invertednormal can correct me here, but I believe the three surface materials with the highest weight values at every point will be blended together with relative weighting to add up to 100%. So for example, if grass has 0.25, dirt has 0.125, sand has 0.125, and rock has 0.05, then the blend will ignore rock, and the result will be 50% grass, 25% dirt, and 25% sand.

Reply:
Sure thing I can extend the description to include this example!